src/Misc/layoutbin/containerHandlerInvoker.js.template

```diff
@@ -18,13 +19,37 @@ process.stdin.on('end', function () {
         process.env[key] = stdinData.environment[key];
```

Comment:
This is where the PATH environment variable is being set. Before the prepend logic would be able to take affect.